t/admin/routes.t

```diff
@@ -2180,3 +2180,65 @@ GET /t
 --- error_code: 400
 --- no_error_log
 [error]
+
+
+
+=== TEST 60: set empty service. (id: 1)（allow empty `service` object）
```

Comment:
we can not do `E2E` test in folder `t/admin/*`.

please move those test case to `t/node/`, `service-empty.t` is a good test file name.

Reply:
ok.